backend/server.js

```diff
@@ -30,9 +30,15 @@ app.get("/working-hours", (req, res) => {
   );
 });
 
-app.get("/monthly-working-hours", (req, res) => {
+app.get("/monthly-working-hours/:year/:amount", (req, res) => {
+  let year = req.params.year;
+  const amount = req.params.amount;
+  let lastYear = Number(year) - Number(amount);
+  year = year.toString();
+  lastYear = lastYear.toString();
   db.all(
-    "SELECT strftime('%Y', date) AS year, strftime('%m', date) AS month, SUM(worker_hours) AS total_hours FROM worklog_worklog GROUP BY year, month ORDER BY year, month",
+    "SELECT strftime('%Y', date) AS year, strftime('%m', date) AS month, SUM(hours) AS total_hours FROM worklog_worklog WHERE (strftime('%Y', date) between ? AND ?) AND (deleted = 0) GROUP BY year, month ORDER BY year, month DESC",
```

Comment:
Year and month as strings is a bit weird, but seems to work fine, so we can leave it as is for now.